Sources/Plasma/Apps/plClient/Mac-Cocoa/PLSLoginWindowController.mm

```diff
@@ -238,8 +111,8 @@ - (IBAction)loginButtonHit:(id)sender
 
     [self.loginParameters makeCurrent];
 
-    [PLSLoginController attemptLogin:^(ENetError result) {
-        [self loginAttemptEndedWithResult:result];
+    [PLSLoginController attemptLogin:^(int result) {
```

Comment:
This is a funny looking change - but Swift doesn't support C++ by default. There is new C++ support, but it can be finicky and I'm not eager to start exposing Plasma headers.  So for now I'm anonymizing ENetError as an int.